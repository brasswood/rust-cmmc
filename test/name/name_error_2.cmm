void main() {
	int a;
	int b;
	void c;
	ptr void d;
	f(a, b);
	g(c, d);
}

void f(int a, int a) {
	# do we enter the scope after encountering an error?
	return hehe;
}

void g(void b, ptr void b) {

}


