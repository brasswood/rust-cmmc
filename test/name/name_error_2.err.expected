FATAL [4,2]: Invalid type in declaration
FATAL [6,2]: Undeclared identifier
FATAL [7,2]: Undeclared identifier
FATAL [7,4]: Undeclared identifier
FATAL [10,19]: Multiply declared identifier
FATAL [12,9]: Undeclared identifier
FATAL [15,8]: Invalid type in declaration
Name Analysis Failed
